### DISCUSSION TITLE
Notice Types CEI, T01, T02, X01 and X02

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#646] [Q&A] [answered]
Hi, 

Where can i get detailed informations (technical and legal) about these notice types CEI, T01, T02, X01 and X02 regarding to eNotices2. 
It would be great if someone can tell me some links. 

Thank you.

Best,
Günter

**Comment by ferrakaop** [comment, marked as answer]
Hello, this question would be more suitable for the TED Helpdesk, as it's not directly related to the SDK.

But in case anyone is interested, these 5 forms are not part of the eForms regulation as they don't implement the procurement directives. T01 and T02 are for public passenger transport services, X01 and X02 are for registration of European businesses and CEI is reserved for EU institutions - see the legal bases at https://enotices2.ted.europa.eu/about-us-news

The forms are included in the SDK as they are part of the implementation of the new systems at the EU Publications Office but if you are only implementing eForms (and the 2014 procurement or 2009 defence directives) then you can safely ignore them.

**Comment by rouschr** [comment]
Hello Günter (Hallo),

for technical or legal questions regarding enotices2 please contact the helpdesk of enotices2: 
https://enotices2.ted.europa.eu/help/contact

If you have specific technical questions regarding the SDK notice types JSON files you can ask them in the SDK github discussions.

SDK documentation about the notice sub types:
https://docs.ted.europa.eu/eforms/latest/notice-types/index.html

The SDK notice sub types are found here (for SDK 1.8):
https://github.com/OP-TED/eForms-SDK/tree/1.8.0/notice-types

**Reply by lepo** [reply]
Hi, 

Thank you for your reply.
We are already using the links for getting all the informations for our development. 
The list of notice sub types includes these types and thats the reason why this question was coming up.

Best, Günter
